blues/application/tasks.py

```diff
@@ -190,3 +191,23 @@ def generate_nginx_conf(role='www'):
 
     with open(conf_path, 'w+') as f:
         f.write(conf)
+
+def notify_deploy(role=None):
```

Comment:
Top-level functions and class definitions should have 2 blank lines before them, while class methods just 1, see https://www.python.org/dev/peps/pep-0008/#blank-lines